draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -44,7 +45,9 @@ export type GuidanceBlockProps = {
     title: string
     description: string | React.ReactNode
   }
+  mobileTextAlignment?: TextAlignment
```

Comment:
Is this really only applicable to mobile? If we just happen to have multiple thinner columns on a desktop, does that mean this will not have an effect? From looking at Figma I don't see that context being a rule (but if it is, then sure).

Reply:
Yup. Based on the tile design we need to provide centre and left alignment specifically for mobile - this is a design requirement.

![Screen Shot 2022-01-18 at 2 10 11 pm](https://user-images.githubusercontent.com/26860255/149864270-960ab935-984a-4e14-bf0e-171d7db92885.png)

 Since we're adjusting the layout/styling based on the guidance block size as opposed to media queries, when the block becomes thin enough it will have the desired effect.

Reply:
That...still sounds like it's not exclusively for mobile then? It's just for when the block is a smaller size? Or am I misunderstanding something?

Reply:
This is when the card reaches a size that's small enough for it to use the designated "mobile" design from the Tile design hence the naming.

Reply:
I think @HeartSquared is right here. Our language tries not to be devices specific and more based on the size of the viewport - although the old mixins are like that, it's hard not to follow suit. 

Ideally we'd use the word `small` as laid out in the UI Kit: https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=23035%3A118325